.github/workflows/ci-multiarch.yaml

```diff
@@ -111,5 +111,5 @@ jobs:
             echo "[!] The job 'build-images' didn't provide any outputs. Can't create the manifest list."
             exit 1;
           fi
-          docker manifest create ${{ env.IMAGE }}:multiarch-next $AMEND
-          docker manifest push ${{ env.IMAGE }}:multiarch-next
+          docker manifest create ${{ env.IMAGE }}:next $AMEND
```

Comment:
it could lead to case where we push manifest with e.g. PPC64LE only.
We need to make all arch required, or all except s390x (s390x sorry).

Maybe something like https://github.com/eclipse/che-dashboard/commit/be438ae369a6797a09e987236b5e5b26f057416f will work but I did not test.

Reply:
just removing `continue-on-error: true` may be alternative approach https://github.com/eclipse/che-dashboard/pull/177/files#diff-0cd88fa8de73cf3bb8b3a02015ac5ff103a7b5a4ea442e75bab856fe9a51ea08L27

Reply:
disabled continue-on-error